Formula/eigensoft.rb

```diff
@@ -0,0 +1,84 @@
+class Eigensoft < Formula
+  # cite Patterson_2006: "https://doi.org/10.1371/journal.pgen.0020190"
+  # cite Price_2006: "https://doi.org/10.1038/ng1847"
+  desc "Eigen tools by Nick Patterson and Alkes Price lab"
+  homepage "https://github.com/DReichLab/EIG"
+  url "https://github.com/DReichLab/EIG/archive/refs/tags/v8.0.0.tar.gz"
+  sha256 "e3459e8ac0134da369910454854eae5c7b261e8816318ccbd2d371b4c6c35741"
+  head "https://github.com/DReichLab/EIG.git"
+
+  depends_on "openblas"
```

Comment:
```suggestion
  depends_on "gsl"
  depends_on "openblas"
```

To solve the linkage error, add 'depends_on "gsl"'.